formation.json

```diff
@@ -171,6 +174,29 @@
           }
         }
       },
+      "ConsolePrivateRackTelemetryQueue": {
+        "Type": "AWS::SQS::Queue",
+        "Properties": {
+          "FifoQueue": "true"
+        }
+      },
+      "ConsolePrivateRackTelemetryQueuePolicy": {
```

Comment:
I think this queue should be created only within our convox infrastructure. Enterprise users won't have access to redshift.
We create the queue in our infra and then in this manifest we give send permission to the console user.
We then process the message inside convox infrasturcture.